src/Common/FiberStack.h

```diff
@@ -37,7 +37,7 @@ class FiberStack
 
     explicit FiberStack(size_t stack_size_ = default_stack_size) : stack_size(stack_size_)
     {
-        ssize_t tmp_page_size = ::sysconf(_SC_PAGESIZE);
+        ssize_t tmp_page_size = getPageSize();
         page_size = (tmp_page_size == -1) ? default_page_size : tmp_page_size;
```

Comment:
It may be better to move the check logic into the getPageSize() function?

Reply:
Hi @bkuschel, Thanks for suggestion, moved the check to getPageSize function.